guacamole/src/main/frontend/src/app/clipboard/templates/guacClipboard.html

```diff
@@ -1 +1,4 @@
-<textarea class="clipboard"></textarea>
+<div>
+    <textarea ng-model="clipboardData" ng-change="updateClipboardData(this.value)" ng-show="isActive" class="clipboard"></textarea>
+    <textarea ng-focus="isActive=true" ng-show="!isActive" class="clipboard inactive">{{'CLIENT.TEXT_CLIPBOARD_AWAITING_FOCUS' | translate}}</textarea>
```

Comment:
This mostly looks good, though I'm not a big fan of directly setting scope variables like this inside the the template like this: `ng-focus="isActive=true"`.

This looks like it's moving controller logic a bit too far into the template, sort of the opposite problem as before. If you created a `setActive()` function or similar on the controller scope, that would probably restore the balance, so to speak.